### DISCUSSION TITLE
Disable HTTP OPTION Method

### DISCUSSION THREAD

**Discussion by rakeshsujan** [netbox-community/netbox#15543] [Help Wanted!]
Hello ALL,

Recently our vulnerability scanner detected to disabled the HTTP OPTION method in our Netbox server.

Can someone help me on how to disable HTTP OPTION method?

**Comment by a084ed22** [comment]
That's something you'll want to do on the reverse proxy

**Comment by rakeshsujan** [comment]
Do you mean in the Nginx? 

If yes can you tell me what entry need to make and in which configuration file?
